Upcoming-iCal-Events.widget/main.coffee

```diff
@@ -98,7 +97,10 @@ update: (output, domEl) ->
         if (lines[i][0] == bullet)
             nameAndCalendar = lines[i].split('(')
             name = nameAndCalendar[0].replace(bullet, '')
-            calendar = nameAndCalendar[1].replace(')','')
+            calendar = 'calendar'
```

Comment:
Could we set this to be 'No Calendar" since it may be output to the user 😄

Reply:
oh，sorry    ：）
 i`ll change that.